gradio/external.py

```diff
@@ -592,15 +592,17 @@ def load_chat(
     *,
     system_message: str | None = None,
     streaming: bool = True,
+    **kwargs,
 ) -> ChatInterface:
     """
     Load a chat interface from an OpenAI API chat compatible endpoint.
     Parameters:
-        base_url: The base URL of the endpoint.
-        model: The model name.
-        token: The API token.
-        system_message: The system message for the conversation, if any.
+        base_url: The base URL of the endpoint, e.g. "http://localhost:11434/v1/"
+        model: The name of the model you are loading, e.g. "llama3.2"
+        token: The API token or a placeholder string if you are using a local model, e.g. "ollama"
```

Comment:
is a placeholder needed for ollama? pretty sure it works without setting any token for ollama

Reply:
It doesn’t actually hence #10307